db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -7852,7 +7865,11 @@ public void faultInTabInfo(TabInfoImpl ti) throws StandardException{
             // Just return in this case, so we don't get a null pointer
             // exception.
             if(td==null){
-                return;
+                // look up from SYSIBM schema
+                td=getTableDescriptor(ti.getTableName(),getSysIBMSchemaDescriptor(),null);
+                if ( td == null) {
```

Comment:
what's is `SysIBMSchema` and why do we search it if we can't find the table in the `SystemSchema`?

Reply:
SYSIBM schema is a legacy system schema inherited from Derby. It is not very useful, but for completeness, splice should also search for this schema, and populate TabInfoImpl for sysibm.sysdummy1 table.